tests/safe_transmute_many/mod.rs

```diff
@@ -0,0 +1,36 @@
+use safe_transmute::{ErrorReason, GuardError, Error, safe_transmute_to_bytes, safe_transmute_many, SingleManyGuard};
+
+type G = SingleManyGuard;
```

Comment:
Why?

Reply:
Whelp, it makes everything less verbose. It's not a bad pattern to employ when the user is only interested in a specific guard in the entire module or code base (which will likely be the case).

Reply:
I mean, like, for end-users, sure.
But in tests? Explicit and verbose is good, IMHO. Especially if the type alias is completely anonymous (even sth like `SMG` would be marginally better)